pkg/builtin/registry/registry.go

```diff
@@ -84,6 +84,7 @@ func newCallCtx(io util.IOStreams, data map[string]interface{}) CallCtx {
 	}
 }
 
+// Run executes tasks
 // Deprecated: Run is deprecated, you should use DoTasks is builtin package, it will automatically register all internal functions
```

Comment:
remove the Deprecated mark is OK, but add an annotation that tell users they should import the builtin package if directly use it

Reply:
got it.